irohad/model/execution/impl/command_executor.cpp

```diff
@@ -322,13 +310,13 @@ namespace iroha {
         auto new_balance =
             account_asset_value.balance + add_asset_quantity.amount;
         if (not new_balance.has_value()) {
-          return makeExecutionError("amount overflows balance");
+          return makeExecutionResult(std::string("amount overflows balance"));
```

Comment:
[`operator""s`](http://en.cppreference.com/w/cpp/string/basic_string/operator%22%22s) may be used instead

Reply:
Did not know about this feature :) Will do